<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16001</width>
     <height>16000</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="PaintArea" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Tree" name="treeWidget">
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: grey</string>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="actionC_ut"/>
    <addaction name="action_Past"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menu123">
    <property name="geometry">
     <rect>
      <x>407</x>
      <y>100</y>
      <width>183</width>
      <height>138</height>
     </rect>
    </property>
    <property name="title">
     <string>Primitivs</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionTriangle"/>
   </widget>
   <widget class="QMenu" name="menuColor">
    <property name="title">
     <string>Color</string>
    </property>
    <addaction name="actionPen"/>
    <addaction name="actionBrush"/>
   </widget>
   <widget class="QMenu" name="menuGrouping">
    <property name="title">
     <string>Grouping</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionGroup"/>
    <addaction name="actionUngroup"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu123"/>
   <addaction name="menuColor"/>
   <addaction name="menuGrouping"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save &amp;as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionC_ut">
   <property name="text">
    <string>C&amp;ut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_Past">
   <property name="text">
    <string>&amp;Past</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="text">
    <string>Line</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="text">
    <string>Ellipse</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="text">
    <string>Triangle</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="text">
    <string>Pen</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="text">
    <string>Brush</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="text">
    <string>&amp;Group</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionUngroup">
   <property name="text">
    <string>&amp;Ungroup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintArea</class>
   <extends>QLabel</extends>
   <header>paintarea.h</header>
  </customwidget>
  <customwidget>
   <class>Tree</class>
   <extends>QTreeWidget</extends>
   <header>tree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
